package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jimple",
-  "version": "1.1.1",
+  "version": "1.2",
```

Comment:
Hello.

I think that this version should be 1.2.0 so npm can validate it.

Can you update it, please?